hashicorp/web-unified-docs
# chore: remove non Terraform content

# Changes
- Remove all content and references to non-tf repos

[Asana task](https://app.asana.com/0/1207899860738460/1208523932117432/f)

# Testing

1. Make a curl request to `https://web-presence-experimental-docs.vercel.app/api/content/nomad/nav-data/latest/docs` (or use Postman)
1. You should need a result similar to this:
<img width="1120" alt="Screenshot 2024-10-16 at 13 53 42" src="https://github.com/user-attachments/assets/e858dd7b-b0aa-4619-aa76-c20724ed45af">
    
1. Make a curl request to `https://web-presence-experimental-docs-git-heat-chorer-edd47a-hashicorp.vercel.app/api/content/nomad/nav-data/latest/docs`
1. You should need a result similar to this:
```bash
Not found
```
